pkg/util/cgroups/cgroupv1_pids.go

```diff
@@ -21,13 +21,13 @@ func (c *cgroupV1) GetPIDStats(stats *PIDStats) error {
 
 	stats.PIDs = nil
 	if err := parseFile(c.fr, c.pathFor("pids", "cgroup.procs"), func(s string) error {
-		pid, err := strconv.ParseInt(s, 10, 64)
+		pid, err := strconv.Atoi(s)
```

Comment:
https://github.com/stephengroat/datadog-agent/security/code-scanning/269?query=ref%3Arefs%2Fpull%2F1%2Fmerge

Reply:
Let's ask Vincent Boulineau about this -- I suspect it's better to just use a known-size value for pids (uint32, I think).